examples/lines_bars_and_markers/errorbar_subsample.py

```diff
@@ -12,22 +12,29 @@
 
 # example data
 x = np.arange(0.1, 4, 0.1)
-y = np.exp(-x)
+y = np.exp(np.vstack([-1.0 * x, -.5 * x]))
 
 # example variable error bar values
-yerr = 0.1 + 0.1 * np.sqrt(x)
+yerr = 0.1 + 0.1 * np.sqrt(np.vstack([x, x/2]))
 
 
 # Now switch to a more OO interface to exercise more features.
-fig, axs = plt.subplots(nrows=1, ncols=2, sharex=True)
+fig, axs = plt.subplots(nrows=1, ncols=3, sharex=True, figsize=(12, 6))
```

Comment:
```suggestion
fig, (ax1, ax2) = plt.subplots(nrows=1, ncols=3, sharex=True, figsize=(12, 6))
```

and using `ax1`, `ax2` below is better than reassigning `ax = ax[i]` later.